### PR TITLE
doc: add openstack on freebsd 2024q3 report

### PR DESCRIPTION
**Pull request by starbops**

### CODE REVIEW

**Review by pauamma — APPROVED** [review]

File: website/content/en/status/report-2024-07-2024-09/openstack.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+In August, we gave link:https://coscup.org/2024/en/session/CKQC9P[a presentation at COSCUP 2024] to share the project's progress and our experiences.
+This helped us get more attention and interest from people in the community.
+
+We also updated some of the OpenStack components, like Keystone, Glance, and Placement, from FreeBSD 14.0-STABLE to FreeBSD 15.0-CURRENT.
+This update helps us keep up with the latest changes in FreeBSD, making the project run better.
+
+Another notable item was testing the link:https://cgit.FreeBSD.org/src/commit/?id=1f903953fbf8615bb611db059417177f6cee07bd[bhyve serial console over TCP patch] and using it in the OpenStack workflow.
+This brings us closer to stopping the use of the custom socat-manager solution and moving to a built-in serial console solution.
+
+Although we are still planning to turn the OpenStack manual installation process into FreeBSD ports, there hasn't been much progress yet.
```
Comment (lsalvadore) [inline website/content/en/status/report-2024-07-2024-09/openstack.adoc:26]: Please avoid contractions.

```suggestion
Although we are still planning to turn the OpenStack manual installation process into FreeBSD ports, there has not been much progress yet.
```

**Review by lsalvadore — APPROVED** [review]
Approved with one small nit.

File: website/content/en/status/report-2024-07-2024-09/openstack.adoc (hunk 1 above)
Comment (starbops) [reply website/content/en/status/report-2024-07-2024-09/openstack.adoc]: Updated, thank you!

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**